app/controllers/users/invitations_controller.rb

```diff
@@ -185,12 +198,18 @@ def check_captcha_for_invite
     def check_invite_users_permission
       @user = current_user
       @emails = params[:emails]&.map(&:downcase)
-      @team = Team.find_by_id(params['teamId'])
+
+      if params['teamId']
```

Comment:
shouldn't it be 'team_ids'?

Reply:
this is for the case of single team invite modal, from the teams section in admin